src/v0/destinations/mp/util.js

```
@@ -257,59 +260,62 @@ const batchEvents = (successRespList, maxBatchSize, reqMetadata) => {
 
-    if (isDefinedAndNotNullAndNotEmpty(traitsValue)) {
-      setOnceEligible[propName] = traitsValue;
+    if (isDefinedNotNullNotEmpty(traitsValue)) {
+      operationEligibleProperties[propName] = traitsValue;
       lodash.unset(traitsCopy, propertyPath);
     }
-    if (isDefinedAndNotNullAndNotEmpty(contextTraitsValue)) {
-      if (!setOnceEligible.hasOwnProperty(propName)) {
-        setOnceEligible[propName] = contextTraitsValue;
+    if (isDefinedNotNullNotEmpty(contextTraitsValue)) {
+      if (!operationEligibleProperties.hasOwnProperty(propName)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)